pkg/defaulting/images.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2012 Datadog, Inc.
```

Comment:
`2012` in the copyright 👀 
Same for the file created in this PR: `pkg/defaulting/images_test.go`.